### DISCUSSION TITLE
Metadata character encodings

### DISCUSSION THREAD

**Discussion by dcabri** [MusicPlayerDaemon/MPD#2150] [Q&A] [answered]
Would it be possible to integrate character encodings other than us-ascii?
I'm gettin many radio strings in iso-8859-1
Or at lease to get the original metadata strings, so I can use iconv on them myself?

**Comment by MaxKellermann** [comment, marked as answer]
MPD has code for character set conversion, but MPD needs to know the character set of a certain string to be able to convert it.
And no, you won't get the original strings. That is a path that leads to sorrow. You can patch MPD to do that, but it won't be a robust solution, and will not be merged here for good reasons. MPD needs to be the one central instance that manages all the strings in a canonical character set.

**Reply by dcabri** [reply]
Indeed! Thank you for pointing this out to me!
I simply had to use
url="${url}#charset=ISO-8859-1"

Thank you so much! (I needed 2 days searching and reading for this ;) )
